amplify/backend/api/colonycdapp/resolvers/Query.searchColonyActions.req.vtl

```diff
@@ -0,0 +1,105 @@
+#set( $args = $util.defaultIfNull($ctx.stash.transformedArgs, $ctx.args) )
```

Comment:
What's the change in this file? As opposed to the original version of this resolver

Reply:
Line 96, the `$args.nextToken` bit wasn't wrapped in double quotes which caused the query to fail